sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -256,6 +256,14 @@ def stage_job_resources(self,
                 'The file "%s" cannot be found. Its location was specified by '
                 'the --sdk_location command-line option.' % sdk_path)
 
+    worker_options = options.view_as(WorkerOptions)
+    if hasattr(worker_options, 'dataflow_worker_jar') and \
```

Comment:
Could you please use ```getattr``` rather than ```hasatter``` :  https://hynek.me/articles/hasattr/

Reply:
interesting read.  However in this particular case (pipelineOption, which is nothing more than just a data field) we are completely safe?    Thoughts? 

That being said, I will apply it for this PR anyway.